### DISCUSSION TITLE
Python Developer Wants to Help

### DISCUSSION THREAD

**Discussion by Clayjoelo** [ONEARMY/community-platform#2422] [General]
I am not sure if this is the place to put it but if you need a Python developer to help out with a project, I can try. I want to support this project but Python is not in the stack. I also know some HTML and CSS if that helps. I'm sorry if this is unwanted or unneeded just thought it wouldn't hurt to say.

**Comment by davehakkens** [comment]
hey thanks for wanting to help @Clayjoelo 
But indeed no Python here atm :(
